pages/02.Setup/05.maxmind-license/docs.en.md

```diff
@@ -21,17 +21,23 @@ From the [2.16 release][216-release], Mautic has supported using a license key t
 Follow these steps to configure your Mautic instance to use the license key.
 
 1. Create a MaxMind account by going to [Maxmind Signup][maxmind-signup]
-1. After logging in, go to "My License Key" on the left hand side in the menu
-![](maxmind-generate-key.png)
-3. Click "Generate new License Key". Answer "Will this key be used for GeoIP Update?" with No
-![](maxmind-generate-key-2.png)
+1. After signup, check your email and click on access your [Maxmind Account][maxmind-account].
+1. Click on the Contact icon at the top right of the menu to login 
+![mautic-maxmind-account](mautic-maxmind-account.png "mautic-maxmind-account")
+1. After logging in, under services click on "My License Key" on the left hand side in the menu
+![maxmind-license-key-2](maxmind-license-key-2.png "maxmind-license-key-2")
+3. Then, Click on Generate new License Key
+![maxmind-generate-key-2](maxmind-generate-key-2.png "maxmind-generate-key-2")
+1. Answer "Will this key be used for GeoIP Update?" with No and confirm
+![maxmind-confirm-key](maxmind-confirm-key.png "maxmind-confirm-key")
 1. Copy the license key that you see on the screen
-![](maxmind-license-key.png)
-1. Go to Mautic > Configuration > System Settings > Miscellaneous Settings and enter the license key into the "IP lookup service authentication" field
-![](mautic-maxmind-license-key.png)
+![maxmind-license-key](maxmind-license-key.png "maxmind-license-key")
+1. Go to Mautic > Settings > Configuration > System Settings > Miscellaneous Settings and enter the license key into the "IP lookup service authentication" field
+![mautic-maxmind-license-key](mautic-maxmind-license-key.png "mautic-maxmind-license-key")
 1. Click "Fetch IP Lookup Data Store". This will download the IP lookup database to your Mautic instance.
 1. Set up the [cron job][cron-jobs] to periodically download a fresh copy.
 
 [216-release]: <https://github.com/mautic/mautic/releases/tag/2.16.0>
+[maxmind-account]: <https://www.maxmind.com/en/accounts/>
 [maxmind-signup]: <https://www.maxmind.com/en/geolite2/signup>
-[cron-jobs]: </setup/cron-jobs>
\ No newline at end of file
+[cron-jobs]: </setup/cron-jobs>
```

Comment:
Small suggestion, it is helpful to use the anchors to drop people at the right part of the page. You can find these by hovering over the headings and clicking or copying the link: https://docs.mautic.org/en/setup/cron-jobs#update-maxmind-geolite2-ip-database.

```suggestion
[cron-jobs]: </setup/cron-jobs#update-maxmind-geolite2-ip-database>
```